src/absil/il.fs

```diff
@@ -3899,8 +3900,41 @@ let decodeILAttribData (ilg: ILGlobals) (ca: ILAttribute) =
       let v, sigptr = parseVal ty sigptr
       parseNamed ((nm, ty, isProp, v) :: acc) (n-1) sigptr
     let named = parseNamed [] (int nnamed) sigptr
-    fixedArgs, named     
-
+    fixedArgs, named
+    
+[<Literal>]
```

Comment:
Add a comment to this explaining things.  Also no need to make it a literal nor reveal it in the signature, it's better to keep it as clean F# code without use of corner-case interop features like `[<Literal>]`

Reply:
I'll see if I can do this without having to expose the literal.